src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/perform_interaction_request.cc

```diff
@@ -880,7 +888,10 @@ bool PerformInteractionRequest::IsWhiteSpaceExist() {
 void PerformInteractionRequest::TerminatePerformInteraction() {
   SDL_LOG_AUTO_TRACE();
 
-  SendClosePopupRequestToHMI();
+  if (mobile_apis::InteractionMode::VR_ONLY != interaction_mode_) {
```

Comment:
```suggestion
  if (!ui_response_received_) {
```
Seems like this would be a better check in this case, since the issue is that this message is being sent when the HMI has already responded to the UI message

Reply:
Done in d0ea3818b99115c4a00cb11bfe193f7d1fe7dbc5

Reply:
Please note that we are performing the regression testing after the latest updates. I’ll let you know when the status will be known

Reply:
@jacobkeeler This PR is ready for review